scw/config.go

```diff
@@ -149,13 +195,29 @@ func (c *Config) Save() error {
 	return c.SaveTo(GetConfigPath())
 }
 
+// HumanConfig will generate a config file with documented arguments.
+func (c *Config) HumanConfig() (string, error) {
+	tmpl, err := template.New("configuration").Parse(configFileTemplate)
+	if err != nil {
+		return "", err
+	}
+
+	var buf bytes.Buffer
+	err = tmpl.Execute(&buf, c)
+	if err != nil {
+		return "", err
+	}
+
+	return buf.String(), nil
```

Comment:
nit: return `.Bytes()` and don't cast on line `232`

Reply:
I think it is more helpful to have String everywhere, in particular for the test. @jerome-quere what do you think ?

Reply:
Also it is a YAML that will not have binary content, therefore string looks like  the best choice